docs/topics/item-pipeline.rst

```diff
@@ -112,7 +119,7 @@ format::
            self.file.close()
 
        def process_item(self, item, spider):
-           line = json.dumps(dict(item)) + "\n"
+           line = json.dumps(dict(ItemAdapter(item))) + "\n"
```

Comment:
I know I pushed for this pattern, but it may be not future proof - see https://github.com/scrapy/itemadapter/issues/27.